iphone/Classes/TiUIToolbar.h

```diff
@@ -1,24 +1,25 @@
 /**
  * Appcelerator Titanium Mobile
- * Copyright (c) 2009-2010 by Appcelerator, Inc. All Rights Reserved.
+ * Copyright (c) 2009-2017 by Appcelerator, Inc. All Rights Reserved.
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
 
 #ifdef USE_TI_UIIOSTOOLBAR
```

Comment:
Actually, we should use `#if defined(USE_TI_UIIOSTOOLBAR) || defined(USE_TI_UITOOLBAR)` all around the place, since Ti.UI.iOS.Toolbar will invoke TiUIToolbarProxy internally as well.